easysocial/easysocial/search.php

```diff
@@ -304,6 +314,73 @@ public function getGroupList($log_user, $search, $limitstart, $limit)
 		return $mapp->mapItem($group, 'group', $log_user->id);
 	}
 
+	// Fetch Pages data
+	/**
+	 * Method Format page object into required object
+	 *
+	 * @param   object   $log_user    user object
+	 * @param   string   $search      search keyword
+	 * @param   integer  $limitstart  limitstart
+	 * @param   integer  $limit       limit
+	 * 
+	 * @return  mixed
+	 *
+	 * @since 1.0
+	 */
+
+	public function getPageList($log_user, $search, $limitstart, $limit)
+	{
+		$mapp = new EasySocialApiMappingHelper;
+		$res = new stdClass;
+		$page = array();
+
+		$db = JFactory::getDbo();
+
+		$query = $db->getQuery(true);
```

Comment:
@UmeshBhosale  - why there's a need to write a query. Can't any model method be used here.